backend/apps/monitoring/models.py

```
@@ -0,0 +1,22 @@
+from django.db import models
+from django.utils import timezone
+
+
+class BackupVerification(models.Model):
+    STATUS_CHOICES = [
+        ("success", "Success"),
+        ("failed", "Failed"),
+    ]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)